src/index.scss

```diff
@@ -1,3 +1,27 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import-normalize;
+
+@import '../node_modules/material-components-web/dist/material-components-web.css';
+
+* {
```

Comment:
WDYT about moving this above the `material-components-web.css` import, just in case some styles in `material-components-web.css` rely on box-sizing?

Reply:
Done.